core/src/mindustry/entities/abilities/MoveLightningAbility.java

```diff
@@ -34,23 +34,27 @@
     public Effect shootEffect = Fx.sparkShoot;
     public Sound shootSound = Sounds.spark;
     
+    protected float side = 1f;
+    
     MoveLightningAbility(){}
     
-    public MoveLightningAbility(float damage, int length, float chance, float offset, float minSpeed, float maxSpeed, Color color, String heatRegion){
+    public MoveLightningAbility(float damage, int length, float chance, float offsetX, float offset, float minSpeed, float maxSpeed, Color color, String heatRegion){
```

Comment:
Again, don't add it to the constructors! This breaks compatibility.

Reply:
Oh wait, now I get what you mean